cpu/esp32/Makefile.dep

```diff
@@ -120,6 +131,14 @@ ifneq (,$(filter mtd,$(USEMODULE)))
   USEMODULE += esp_idf_spi_flash
 endif
 
+ifneq (,$(filter nimble,$(USEPKG)))
```

Comment:
```suggestion
ifneq (,$(filter esp_ble_nimble,$(USEPKG)))
```

Reply:
This conditional is correct, `esp_ble_nimble` is only a pseudomodule which is enabled if package `nimble` is used.